metricbeat/mb/module/connector.go

```diff
@@ -70,6 +76,17 @@ func NewConnector(
 	}, nil
 }
 
+// UseMetricSetProcessors appends processors defined in metricset configuration to the connector properties.
+func (c *Connector) UseMetricSetProcessors(r metricSetRegister, moduleName, metricSetName string) error {
+	metricSetProcessors, err := r.ProcessorsForMetricSet(moduleName, metricSetName)
+	if err != nil {
+		return errors.Wrapf(err, "reading metricset processors failed (module: %s, metricset: %s)",
+			moduleName, metricSetName)
+	}
+	c.processors.AddProcessors(*metricSetProcessors)
```

Comment:
Processors defined in the manifest should be executed before processors defined in the configuration.

Reply:
Fixed, but please verify if it's alright.